src/js/_store/modules/categories.js

```diff
@@ -0,0 +1,31 @@
+import axios from '@Helpers/axiosDefault';
+
+export default {
+  namespaced: true,
+  state: {
+    categoryList: [],
+    errorMessage: '',
+  },
+  mutations: {
+    doneGetAllCategories(state, categories) {
+      state.categoryList = categories.reverse();
+    },
+    failRequest(state, { message }) {
+      state.errorMessage = message;
+      state.loading = false;
+    },
+  },
+  actions: {
+    getAllCategories({ commit, rootGetters }) {
+      axios(rootGetters['auth/token'])({
+        method: 'GET',
+        url: '/category',
+      }).then(({ data }) => {
+        const categoryList = data.categories;
+        commit('doneGetAllCategories', categoryList);
```

Comment:
[nits]

一時変数は逆に可読性を落とすので、1度しか使わない場合、そのまま使ってあげたほうが読みやすいです。

```suggestion
        commit('doneGetAllCategories', data.categories);
```

Reply:
了解しました。